### PR TITLE
feat(cdp): add loops hog function

### PR DESCRIPTION
**Pull request by MarconLP**
## Changes

- [X] add loops hog function
- [x] add tests

Changes compared to the old plugin:
- we'll always send person properties compared to only on `$identify` and `$set` events.

## How did you test this code?

compare the webhook payload to the [old loops plugin](https://github.com/PostHog/posthog-loops-plugin)

### CODE REVIEW

File: posthog/cdp/templates/loops/template_loops.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+fetch('https://app.loops.so/api/v1/events/send', {
+    'method': 'POST',
+    'headers': {
+        'Content-Type': 'application/json',
+        'Authorization': f'Bearer {apiKey}',
+    },
+    'body': payload
+})
+""".strip(),
+    inputs_schema=[
```
Comment (benjackwhite) [inline posthog/cdp/templates/loops/template_loops.py:30]: this feels like a "person updates" plugin like some of the others so can we add the filters for $identify as a sensible default

**Review by benjackwhite — CHANGES_REQUESTED** [review]

File: posthog/cdp/templates/loops/test_template_loops.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+from inline_snapshot import snapshot
+from posthog.cdp.templates.helpers import BaseHogFunctionTemplateTest
+from posthog.cdp.templates.loops.template_loops import template as template_loops
+
+
+class TestTemplateLoops(BaseHogFunctionTemplateTest):
+    template = template_loops
+
+    def _inputs(self, **kwargs):
+        inputs = {"apiKey": "1cac089e00a708680bdb1ed9f082d5bf"}
```
Comment (benjackwhite) [inline posthog/cdp/templates/loops/test_template_loops.py:10]: Just checking - this isn't a real api key right?

**Review by benjackwhite — APPROVED** [review]
Looks good!

File: posthog/cdp/templates/loops/test_template_loops.py (hunk 2 above)
Comment (MarconLP) [reply posthog/cdp/templates/loops/test_template_loops.py:10]: It was until I disabled that API key.

**Pull request merged**